modules/ROOT/pages/reproducible-builds.adoc

```diff
@@ -0,0 +1,11 @@
+= Reproducible Builds
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You should be able to build a particular version of your Mule project at any point of time. This facilitates maintenance as the versions of your projects in production will certainly not be the latest ones.
```

Comment:
Re "should":  I believe we're supposed to avoid that term and replace it with something more definitive.  

"as" -> "because" or some such